### PR TITLE
DR-1155: Add Perf server config, set perf as default env for tests

### PR DESCRIPTION
**Pull request by snf2ye**

### CODE REVIEW

**Review by fboulnois — APPROVED** [review]
getting perf up and running 🎉

**Review by RoriCremer — APPROVED** [review]
I have a quick q about motivation just cuz I'm curious that I'll bug you about in slack

File: datarepo-clienttests/src/main/resources/servers/perf.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "perf",
+  "description": "Dedicated perf cluster.",
+  "datarepoUri": "https://jade-perf.datarepo-perf.broadinstitute.org",
+  "samUri": "https://sam.dsde-dev.broadinstitute.org",
+  "samResourceIdForDatarepo": "broad-jade-dev",
```
Comment (marikomedlock) [inline datarepo-clienttests/src/main/resources/servers/perf.json:6]: Didn't you setup a new SAM resource just for perf?

**Review by marikomedlock — APPROVED** [review]
LGTM. One question inline about SAM resource id.

**Pull request merged**